frontend/src/components/core/Text.tsx

```diff
@@ -4,6 +4,10 @@ export const Text = styled.p`
   font-size: ${({ theme }) => theme.fontSize.default};
 `;
 
+export const BoldText = styled(Text)`
+  font-weight: bold;
+`;
+
```

Comment:
Thoughts on replacing `BoldText` with `Text` that has a `bold` attribute parameter?

Reply:
Good idea - added an optional bold property to the Text component to allow for this